auction_seller_invoice/views/auction_item_invoice_views.xml

```diff
@@ -0,0 +1,597 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="account_move_seller_form_view" model="ir.ui.view">
```

Comment:
What are the adjustments made from account.view_move_form?  Why can't we inherit it make adjustments?

Reply:
In this view_move_form, I thought seller user should only see settlement, invoice lines, amount . So, If we inherit existing view_move_form and remove some part, that will effect original view_move_form. Thats why I decided to create separate form view with some necessary part.

Reply:
Shouldn't `<field name="mode">primary</field>` do?